ui/components/multichain/account-list-menu/account-list-menu.js

```diff
@@ -83,201 +78,220 @@ export const AccountListMenu = ({ onClose }) => {
   }
 
   return (
-    <Popover
-      title={title}
-      ref={inputRef}
-      centerTitle
-      onClose={onClose}
-      onBack={actionMode === '' ? null : () => setActionMode('')}
-      className="multichain-account-menu-popover"
-    >
-      {actionMode === 'add' ? (
-        <Box paddingLeft={4} paddingRight={4} paddingBottom={4} paddingTop={0}>
-          <CreateAccount
-            onActionComplete={(confirmed) => {
-              if (confirmed) {
-                dispatch(toggleAccountMenu());
-              } else {
-                setActionMode('');
-              }
-            }}
-          />
-        </Box>
-      ) : null}
-      {actionMode === 'import' ? (
-        <Box paddingLeft={4} paddingRight={4} paddingBottom={4} paddingTop={0}>
-          <ImportAccount
-            onActionComplete={(confirmed) => {
-              if (confirmed) {
-                dispatch(toggleAccountMenu());
-              } else {
-                setActionMode('');
-              }
-            }}
-          />
-        </Box>
-      ) : null}
-      {actionMode === '' ? (
-        <Box>
-          {/* Search box */}
-          {accounts.length > 1 ? (
-            <Box
-              paddingLeft={4}
-              paddingRight={4}
-              paddingBottom={4}
-              paddingTop={0}
-            >
-              <TextFieldSearch
-                size={Size.SM}
-                width={BlockSize.Full}
-                placeholder={t('searchAccounts')}
-                value={searchQuery}
-                onChange={(e) => setSearchQuery(e.target.value)}
-                clearButtonOnClick={() => setSearchQuery('')}
-                clearButtonProps={{
-                  size: Size.SM,
-                }}
-              />
-            </Box>
-          ) : null}
-          {/* Account list block */}
-          <Box className="multichain-account-menu-popover__list">
-            {searchResults.length === 0 && searchQuery !== '' ? (
-              <Text
+    <Modal isOpen onClose={onClose}>
+      <ModalOverlay />
+      <ModalContent
+        className="multichain-account-menu-popover"
+        modalDialogProps={{ padding: 0 }}
```

Comment:
I need to adjust this in the component but this should remove the extra scroll bar

```suggestion
        modalDialogProps={{ padding: 0, marginBottom: 0 }}
```

<img width="369" alt="Screenshot 2023-06-28 at 11 16 03 AM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/03b551fc-139c-4928-be7a-48fe8fce2ca9">

<img width="365" alt="Screenshot 2023-06-28 at 11 14 22 AM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/2234ecb1-3576-43f2-a287-bec6351332cf">